src/screen.rs

```diff
@@ -60,23 +76,24 @@ impl Interfaceable for PrettyPrompt {
         println!("{}", string);
     }
 
-    fn prompt(&mut self) -> String {
+    fn prompt(&mut self) -> Result<String, MaeveError> {
         let readline = match self.editor.readline(">> ") {
             Ok(line) => {
                 if self.history {
                     self.editor.add_history_entry(&line);
                 }
-                return line;
+                line
```

Comment:
Hmm. I know this is rusty, but low key less readable (at least to me)

Reply:
So for this you actually have to do it like that otherwise it thinks we're trying to return from the function instead of just assigning the variable. But on a side note, I was gonna see if we might go through and just use this convention for all of our return statements anyway.